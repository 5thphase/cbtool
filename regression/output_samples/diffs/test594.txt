|Redis Version |2.8.6 
|HOSTs |62 
|VMCs ISSUED |18 
|HOSTs ISSUED |0 
|VMs ISSUED |129 
|VMs SLA PROVISIONING VIOLATED |0 
|VMs SLA RUNTIME VIOLATED |0 
|VMs APPLICATION ERRORS |12 
|AIs ISSUED |37 
|AIDRSs ISSUED |2 
|VMCRSs ISSUED |0 
|FIRSs ISSUED |0 
|EXPERIMENT COUNTER |539 
|name |role |size |cloud_ip |host_name |vmc_pool |netname |ai |aidrs |uuid
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |dest |sut |private |none |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
