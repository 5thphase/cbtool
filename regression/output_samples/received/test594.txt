 status: Sending a migrate request for vm_129 (cloud-assigned uuid 83973C7B-3E67-52F9-9DEC-22903CDBC3BE)....
VM object E078879C-76F2-5CED-A385-493D8F5B24A7 (named "vm_129") successfully migrateed on this experiment.
The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |6.66M                         
|Redis Version                                        |2.6.16                        
|Uptime (in seconds)                                  |2218848                       
|Total Connections Received                           |8217                          
|Total Commands Processed                             |2727131                       
|Number of Keys                                       |703                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.6                         
|Storage Size                                         |2805760                       
|Data Size                                            |777248                        
|Index Size                                           |482384                        
|Average Object Size                                  |545.820224719                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |6                             
|HOSTs                                                |22                            
|VMs                                                  |1                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |6                             
|VMCs ARRIVED                                         |18                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |1                             
|VMs ARRIVED                                          |125                           
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |124                           
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |4                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |35                            
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |35                            
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |2                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |2                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |2                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |419                           

 ---------------- VMC-WIDE COUNTERS ----------------
|6B2CF2C9-7ECD-535A-9836-6D30EBA8F83F (simzone_d) VM RESERVATIONS|0                             
|A3B38AF9-68E8-5E7C-8050-66F6D9681E02 (simzone_a) VM RESERVATIONS|0                             
|845D544A-3F3E-5433-BD75-03F09FBB3BE7 (simzone_c) VM RESERVATIONS|0                             
|F1774E78-5022-5568-80E1-9A3EFC329DB1 (simzone_b) VM RESERVATIONS|0                             
|9F08B39F-EDBD-5E00-9AE5-C2F76A29FE52 (simzone_dest) VM RESERVATIONS|1                             
|7B341241-FD86-5B5B-9552-ADC6554C8742 (simzone_source) VM RESERVATIONS|0                             

The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_129        |tinyvm              |pico32      |11.13.92.102    |dest                       |sut                 |none    |none       |E078879C-76F2-5CED-A385-493D8F5B24A7

