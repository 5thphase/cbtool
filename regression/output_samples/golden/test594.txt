 status: Sending a migrate request for vm_129 (cloud-assigned uuid A68EF1D6-C0C6-51DB-8D39-321065B4BE5B)....
VM object 1F2E74A1-0FE9-502A-97C3-94E71B40C2A6 (named "vm_129") successfully migrateed on this experiment.
The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |7.23M                         
|Redis Version                                        |2.5.12                        
|Uptime (in seconds)                                  |2417983                       
|Total Connections Received                           |4653                          
|Total Commands Processed                             |4105900                       
|Number of Keys                                       |703                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.5                         
|Storage Size                                         |5449940992.0                  
|Data Size                                            |4963223464.0                  
|Index Size                                           |133841120                     
|Average Object Size                                  |8098.60742858                 
|Collections                                          |40                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |6                             
|HOSTs                                                |22                            
|VMs                                                  |1                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |6                             
|VMCs ARRIVED                                         |18                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |1                             
|VMs ARRIVED                                          |125                           
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |124                           
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |4                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |35                            
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |35                            
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |2                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |2                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |2                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |419                           

 ---------------- VMC-WIDE COUNTERS ----------------
|CEA17C37-820A-507F-9356-90956267A568 (simzone_a) VM RESERVATIONS|0                             
|966D86CB-90BB-553D-A06A-0FFA0818D568 (simzone_d) VM RESERVATIONS|0                             
|BE03B636-352A-55D9-83EE-A6E863233E00 (simzone_dest) VM RESERVATIONS|1                             
|2FF9728E-95B3-5EE9-A0A4-61875B507FA8 (simzone_b) VM RESERVATIONS|0                             
|36FB7090-0472-5A58-8489-821C791FAF42 (simzone_source) VM RESERVATIONS|0                             
|C41BC369-F411-5382-B95F-70C31BD20E23 (simzone_c) VM RESERVATIONS|0                             

The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_129        |tinyvm              |pico32      |83.93.228.6     |dest                       |sut                 |none    |none       |1F2E74A1-0FE9-502A-97C3-94E71B40C2A6

